reporter/reporter.go

```diff
@@ -228,7 +228,19 @@ func (t *reporter) reportDerivedMetrics(span tracer.RawSpan) {
 	replaceTag(tags, "application", appName, appFound)
 	replaceTag(tags, "service", serviceName, svcFound)
 
-	t.getHistogram(metricName+".duration.micros", tags).Update(span.Duration.Nanoseconds() / 1000)
+	v, found := getAppTag("error", "false", span.Tags)
+	if found && v == "true" {
+		tagsError := t.application.Map()
```

Comment:
Rather than repeat lines 225-229, you can write a helper function that takes in the `tags` map as an argument and returns a copy of the map (make a new map of same length and copy the keys/values). That way if we add more things to the tags map higher up we don't need to change this part of the code.